lang/lang_en_pl.txt

```diff
@@ -1236,8 +1236,200 @@
 
 #
 "Y-correct:"
-"Korekcja Y:"
+"Korekcja-Y:"
 
 #MSG_OFF
 " [off]"
+" [wyl]"
+
+#
+"Back"
+"Wstecz"
+
+#
+"Checks"
+"Testy"
+
+#
+"False triggering"
+"Falszywy alarm"
+
+#
+"FINDA:"
 "\x00"
+
+#
+"Firmware   [none]"
+"Firmware   [brak]"
+
+#
+"Firmware [strict]"
+"Firmware [restr.]"
+
+#
+"Firmware   [warn]"
+"Firmware[ostrzez]"
+
+#
+"HW Setup"
+"Ustawienia HW"
+
+#
+"IR:"
+"\x00"
+
+#
+"Magnets comp.[N/A]"
+"Kor. magnesow [nd]"
+
+#
+"Magnets comp.[Off]"
+"Kor. magnesow[wyl]"
+
+#
+"Magnets comp. [On]"
+"Kor. magnesow [wl]"
+
+#
+"Mesh         [3x3]"
+"Siatka       [3x3]"
+
+#
+"Mesh         [7x7]"
+"Siatka       [7x7]"
+
+#
+"Mesh bed leveling"
+"Poziomowanie wg siatki"
+
+#
+"MK3S firmware detected on MK3 printer"
+"Wykryto firmware MK3S w drukarce MK3"
+
+#
+"MMU Mode [Normal]"
+"Tryb MMU[Normaln]"
+
+#
+"MMU Mode[Stealth]"
+"Tryb MMU[Stealth]"
+
+#
+"Mode change in progress ..."
+"Trwa zmiana trybu..."
+
+#
+"Model      [none]"
+"Model      [brak]"
+
+#
+"Model    [strict]"
+"Model [restrykc.]"
+
+#
+"Model      [warn]"
+"Model  [ostrzez.]"
+
+#
+"Nozzle d.  [0.25]"
+"Sr. dyszy  [0,25]"
+
+#
+"Nozzle d.  [0.40]"
+"Sr. dyszy  [0,40]"
+
+#
+"Nozzle d.  [0.60]"
+"Sr. dyszy  [0,60]"
+
+#
+"Nozzle     [none]"
+"Dysza      [brak]"
+
+#
+"Nozzle   [strict]"
+"Dysza [restrykc.]"
+
+#
+"Nozzle     [warn]"
+"Dysza  [ostrzez.]"
+
+#
+"G-code sliced for a different level. Continue?"
+"\x00"
+
+#
+"G-code sliced for a different level. Please re-slice the model again. Print cancelled."
+"G-code pociety na innym poziomie. Potnij model ponownie. Druk anulowany."
+
+#
+"G-code sliced for a different printer type. Continue?"
+"G-code pociety dla innej drukarki. Kontynuowac?"
+
+#
+"G-code sliced for a different printer type. Please re-slice the model again. Print cancelled."
+"G-code pociety dla drukarki innego typu. Potnij model ponownie. Druk anulowany."
+
+#
+"G-code sliced for a newer firmware. Continue?"
+"G-code pociety dla nowszego firmware. Kontynuowac?"
+
+#
+"G-code sliced for a newer firmware. Please update the firmware. Print cancelled."
+"G-code pociety dla nowszego firmware. Zaktualizuj firmware. Druk anulowany."
+
+#
+"PINDA:"
+"\x00"
+
+#
+"Preheating to cut"
+"Nagrzewanie do obciecia"
+
+#
+"Preheating to eject"
+"Nagrzewanie do wysuniecia"
+
+#
+"Printer nozzle diameter differs from the G-code. Continue?"
+"Srednica dyszy drukarki rozni sie od tej w G-code. Kontynuowac?"
+
+#
+"Printer nozzle diameter differs from the G-code. Please check the value in settings. Print cancelled."
+"Srednica dyszy rozni sie od tej w G-code. Sprawdz ustawienia. Druk anulowany."
+
+#
+"Rename"
+"Zmien nazwe"
+
+#
+"Select"
+"Wybierz"
+
+#
+"Sensor info"
+"Info o sensorach"
+
+#
+"Sheet"
+"Plyta"
+
+#
+"Sound     [assist]"
```

Comment:
After following https://github.com/DRracer/Prusa-Firmware/blob/translations-doc/lang/translations.md the not_tran.txt showed that Sound [assist] is missing.
Checking the lang_en.txt and lang_en_??.txt it seams to be a typo using one space too much.

The message should be 17 chars long and not 18 with 4 spaces between Sound and [assist]